index.html

```diff
@@ -1,16 +1,130 @@
 <!doctype html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <title>Great Idea - Responsive I</title>
+    <head>
+        <meta charset="utf-8">
 
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+        <title>Great Idea!</title>
 
-</head>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+        <link rel="stylesheet" href="css/index.css">
 
```

Comment:
always make sure meta tags and link tags to your fonts and stylesheets are INSIDE the <head></head> tags of your document